docs/sections/user_guide/yaml/tags.rst

```diff
@@ -0,0 +1,76 @@
+.. _defining_YAML_tags:
+
+Defining YAML ``tags``
+==========================
+
+Tags are used to denote the type of a YAML node when converting to a Python object. Standard YAML tags are defined at http://yaml.org/type/index.html.
+
+Tags may be implicit:
+
+.. code-block:: yaml
+
+   boolean: true
+   integer: 3
+   float: 3.14
+
+Or explicit:
+
+.. code-block:: yaml
+
+   boolean: !!bool "true"
+   integer: !!int "3"
+   float: !!float "3.14"
+
+Additionally, UW extends the following tags and constructors to support other formats:
+
+UW YAML Tags
+------------
+
+``!float``
+^^^^^^^^^^
+
+Converts the tagged node to a Python ``float`` type.
+
+.. code-block:: yaml
+
+   f2: !float '3.14'
+
+
+``!int``
+^^^^^^^^
+
+Converts the tagged node to a Python ``integer`` type.
+
+.. code-block:: yaml
+
+   cycle_day: !int "{{ cycle.strftime('%d') }}"
+
+
+``!include``
+^^^^^^^^^^^^
+
+Parse the tagged file and include its tags.
+
+.. code-block:: yaml
+
+   salad: !INCLUDE [./fruit_config.yaml]
+
+
+``!list``
```

Comment:
Does this actually exist?

Reply:
AH - huh, now that you mention it, its in the ticket, but we never made a `!list` tag. As its optional in the AC, I'll remove it for now and we can talk about whether we actually need that